docs/configure/cli/automation-rules.md

```diff
@@ -0,0 +1,37 @@
+# Uploading Rules
```

Comment:
This page is duplicated from the above "automation/rules.md" files. Is that on purpose?